content/docs/deployment.md

```diff
@@ -193,7 +193,7 @@ usercert = ~/.cassandra/client-cert
 
 ### ElasticSearch
 
-Supported versions: 5.x, 6.x
```

Comment:
As people have asked explicitly about 6.x before, I'd add a note saying that "We are aware of problems with 6.x and would welcome contributions in fixing them", linking to https://github.com/jaegertracing/jaeger/issues/665

Reply:
I would rather point them to the https://github.com/jaegertracing/jaeger/issues/637. Which can track various es6 issues